### ISSUE TITLE
Please add a keyboard Shortcut for Zoom in and Out

### ISSUE THREAD

**Issue by Sage222** [xournalpp/xournalpp#5373]
### Feature Description

Please add a keyboard Shortcut for Zoom in and Out.   Using "CTRL +" and "CTRL -"  would bring into line with other application standards.

Thank you for considering my request!

**Comment by rolandlo** [comment]
Those keyboard shortcuts for zoom in and zoom out are already implemented. What Xournal++ version and what OS do you use?

**Comment by Sage222** [comment]
i should add "CTRL +" does work properly on my system with other apps like Chrome. So it does not appear to be my Keyboard or System setup :)

**Comment by rolandlo** [comment]
This looks like another case where we need to mask out consumed modifiers (like most recently in e679dd76c3e27319284f63afaa60750e3c3eee35). We already have it in the text editor, where Ctrl+plus increases font size.

So can you please check if Ctrl+plus increases font size when you use the text tool?

**Comment by Sage222** [comment]
Same thing occurs.  CTRL- reduces the Text Font. Though CTRL+ does not increase it.

**Comment by rolandlo** [comment]
Hmm, what's your keyboard layout then?

**Comment by Sage222** [comment]
US Qwerty Keyboard Layout.

Australian Region Language (Not that should matter) :)

Just tried the keyboard shortcut with a few other apps too.. Notepad++,  Microsoft Teams, Microsoft Edge.  All work perfectly.

**Comment by rolandlo** [comment]
So that mean this layout?
![grafik](https://github.com/xournalpp/xournalpp/assets/40485433/29ca3ea7-2b3d-44ec-9ace-22282c2453a3)

And you press Ctrl+Shift+the key left to the delete key?

**Comment by Sage222** [comment]
yep thats the one. 

here is a pic of the actual keyboard if that helps.

https://p1.akcdn.net/full/765890871.logitech-g-pro-gx-clicky-920-009392.jpg

**Comment by rolandlo** [comment]
Sorry to ask again, but are you really holding the shift key while pressing `<Ctrl>plus` ? Or do you only hit `<Ctrl>=`? When I use the US Qwerty layout, it works with the shift key in Xournal++, whereas in Firefox I don't need to hold the shift key.

**Comment by Sage222** [comment]
Yep definately using CTRL +  as i said it works fine on all other apps.  I just tried to use SHIFT instead and as expected neither Zoom in or Out works with shift.  I just set it all up on my laptop as well. And i have the same problems as my Desktop.

Thinking of a workaround maybe add a Default Zoom level for every document that you open?

**Comment by rolandlo** [comment]
> Yep definately using CTRL + as i said it works fine on all other apps. I just tried to use SHIFT instead and as expected neither Zoom in or Out works with shift. 

I did not suggest to use SHIFT *instead*. I suggested to press CTRL, SHIFT and the 
![grafik](https://github.com/xournalpp/xournalpp/assets/40485433/241592a0-0c92-4a38-968e-bc94f1845d7d)
key *at the same time*.

It's different with CTRL minus. There you should not press the SHIFT key.

When you write a minus (-) or plus (+) in text, you also have to press the SHIFT key only for the plus, since otherwise you get a `=`
![grafik](https://github.com/xournalpp/xournalpp/assets/40485433/107e1423-1c09-4591-ac84-2420ce56c4ee)

Please also check with the text tool in Xournal++.

**Comment by rolandlo** [comment]
Probably the other apps add Ctrl+= as an alternative shortcut. Firefox does as I can see using my Swiss German keyboard layout. This feels a bit odd on non-US keyboards (where the "=" and "+" are on different keys), but maybe doesn't do much harm. So we could consider doing that as well.
